File: KuantumJava/VeriPaketi.java
// güvenli veri nesnesi (IKritik değil)
public class VeriPaketi extends KuantumNesnesi {
    public VeriPaketi(String id) {
        super(id);
    }
    @Override
    public void analizEt() throws KuantumCokusuException {
        setStabilite(getStabilite() - 5);      // stabilite 5 azalır
        System.out.println("Veri içeriği okundu.");
    }
}

File: KuantumCSharp/KuantumOdev/VeriPaketi.cs
using System;

// güvenli veri nesnesi (Kritik değil)
public class VeriPaketi : KuantumNesnesi
{
    public VeriPaketi(string id)
    {
        ID = id;
        Stabilite = 70.0;                           // örnek başlangıç stabilitesi
        TehlikeSeviyesi = new Random().Next(1, 11);  // 1-10 rastgele değer
    }

    public override void AnalizEt()
    {
        Stabilite -= 5;                              // stabilite 5 azalır
        Console.WriteLine("Veri içeriği okundu.");   
    }
}
